electrum/lnchannel.py

```diff
@@ -566,10 +551,13 @@ def is_frozen_for_receiving(self) -> bool:
         return False
 
     def is_static_remotekey_enabled(self) -> bool:
-        # Return False so that self.sweep_address will return self._fallback_sweep_address
+        return False
```

Comment:
why is this False?
Looks like the old comment might not apply anymore.

Reply:
indeed, but it does not hurt. 
that way the asserts will fail if executed on a backup.

Reply:
I would much prefer explicit `assert not self.is_backup()` lines (either in addition to any existing assert or instead of them, depending on what makes sense) wherever that is relevant :)

> indeed, but it does not hurt.

I think it is confusing. If we keep it as False, at the least there should be a code comment here to explain.

Reply:
Also, to be clear, we don't want to support backups of legacy channels (that have static_remotekey disabled), right? -- just to clarify whether that's related to what is returned here

Reply:
since that method is only called for channels, not channel backups, I think it is better to remove it.
ff3c999fcb7befb5c27a0b18d3dfa5fed1b5a18b